charts/drax/configuration/grafana/dashboards/5g-cucp-pm-counters.json

```diff
@@ -110,7 +110,8 @@
             "type": "prometheus",
             "uid": "PBFA97CFB590B2093"
           },
-          "expr": "increase(RRC_CONNECTION_SUCC_total {instanceId=~\"$InstanceID\",instanceType=~\"$instanceType\"} [$__rate_interval]) * 100 / increase(RRC_CONNECTION_EST_ATTEMPT_total {instanceId=~\"$InstanceID\",instanceType=~\"$instanceType\"}[$__rate_interval])",
+          "editorMode": "code",
+          "expr": "round(increase(RRC_CONNECTION_SUCC_total {instanceId=~\"$InstanceID\",instanceType=~\"$instanceType\"}[$MinStep])) * 100 / round(increase(RRC_CONNECTION_EST_ATTEMPT_total {instanceId=~\"$InstanceID\",instanceType=~\"$instanceType\"}[$MinStep]))",
```

Comment:
rate_interval is something introduced by grafana to solve problems related to calculating and graphing rates from Prometheus metrics. As discussed, since we see that MinStep (which is a parameter set by us via the dropdown) fits better our needs, let's go with this.

https://grafana.com/blog/2020/09/28/new-in-grafana-7.2-__rate_interval-for-prometheus-rate-queries-that-just-work/